src/daemon/common_functions.sh

```diff
@@ -141,15 +141,11 @@ function dev_part {
 
 function osd_trying_to_determine_scenario {
   : "${OSD_DEVICE:=none}"
-  if [[ ${OSD_DEVICE} == "none" ]]; then
-    log "Bootstrapped OSD(s) found; using OSD directory"
-    source /opt/ceph-container/bin/osd_directory.sh
-    osd_directory
-  elif parted --script "${OSD_DEVICE}" print | grep -sqE '^ 1.*ceph data'; then
+  if parted --script "${OSD_DEVICE}" print | grep -sqE '^ 1.*ceph data'; then
     log "Bootstrapped OSD found; activating ${OSD_DEVICE}"
     source /opt/ceph-container/bin/osd_disk_activate.sh
     osd_activate
-  else
+  elif [[ ${OSD_DEVICE} != "none" ]]; then
```

Comment:
Why not `else`?

Reply:
because we don't want to trigger the osd_disk function if `${OSD_DEVICE} == "none"`

Reply:
Should there maybe be an `else` error scenario then?  I'm not familiar enough with these scenarios to know if these are the only two options and no other (for lack of a better word) scenario could exist.

Reply:
I guess in other words, is it better than nothing is done or that the function errors out if neither condition is met?

Reply:
both are valid solutions but I guess to make sense to log and exit in this situation